File: seminar_4/Python/task2.py
# Напишите программу, которая принимает на вход число и выдаёт количество цифр в числе.
# 456 -> 3
# 78 -> 2
# 89126 -> 5

def CountNumbers(number):
    result = 0
    while number > 0:
        result += 1
        number //= 10

    return result


num = int(input("Введите число: "))
print(num, CountNumbers(num))

File: seminar_4/Java/task2.java
// Напишите программу, которая принимает на вход число и выдаёт количество цифр в числе.
// 456 -> 3
// 78 -> 2
// 89126 -> 5
package seminar_4.Java;

import java.util.Scanner;

public class task2 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Введите число: ");
        int num = input.nextInt();
        int count = 0;
        while (num > 0) {
            count++;
            num /= 10;
        }
        System.out.println(count);
    }
}
